core/src/test/kotlin/com/alecstrong/sql/psi/core/AnsiSqlTests.kt

```diff
@@ -18,9 +18,7 @@ class AnsiSqlTests(name: String, fixtureRoot: File) : FixturesTest(name, fixture
     "AUTOINCREMENT" to "DEFAULT 0",
   )
 
-  override fun setupDialect() {
-    // No-op.
-  }
+  override fun setupDialect(): List<PredefinedTable> = emptyList()
```

Comment:
i think we should just have a separate method or make this an optional constructor param